### PR TITLE
Jinja2-ize latex writer

### PR DESCRIPTION
**Pull request by tk0miya**
Move LaTeX template to independent file. This allows replacement the template in future version.

### CODE REVIEW

File: sphinx/writers/latex/content.tex_t [hunk 1]
```diff
@@ -0,0 +1,42 @@
+%% Generated by Sphinx.
```
Comment (shimizukawa) [inline sphinx/writers/latex/content.tex_t]: location under `sphinx/templates/` or `sphinx/writers/latex/templates/` would be better. (relates to #2749.)

**Comment by shimizukawa** [comment]
LGTM with nits ;)

File: sphinx/writers/latex/content.tex_t (hunk 1 above)
Comment (tk0miya) [inline sphinx/writers/latex/content.tex_t]: Moved to `sphinx/templates/`.

Comment (tk0miya) [inline sphinx/writers/latex/content.tex_t]: To be precise more, I moved it to `sphinx/templates/latex/content.tex_t`

**Comment by tk0miya** [comment]
Thank you for reviewing!

**Pull request merged**